src/app/pages/entradas/ver-comprobante/ver-comprobante.component.html

```diff
@@ -0,0 +1,116 @@
+<nb-card-header>
+<h1>Comprobante Contable</h1>
+</nb-card-header>
+
+
+      <div class="row">
+        <div class="col">
+          <p class="text-uppercase"><strong>Consecutivo</strong>:{{consecutivo}}</p>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col">
+          <p class="text-uppercase"><strong>Concepto</strong>
+           : {{transaccion.Descripcion}}</p>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col">
+          <p class="text-uppercase"><strong>Fecha Transacción</strong>
+           : {{transaccion.FechaTransaccion}}</p>
+        </div>
+      </div>
+
+
+      <div class="row">
+        <div class="col">
+          <p class="text-uppercase"><strong>Secuencia</strong></p>
+        </div>
+        <div class="col">
+          <p class="text-uppercase"><strong>Tercero</strong></p>
+        </div>
+        <div class="col">
+          <p class="text-uppercase"><strong>Cuenta</strong></p>
+        </div>
+        <div class="col">
+          <p class="text-uppercase"><strong>Nombre Cuenta</strong></p>
+        </div>
+        <div class="col">
+          <p class="text-uppercase"><strong>Descripcion</strong></p>
+        </div>
+        <div class="col">
+          <p class="text-uppercase"><strong>Débito</strong></p>
+        </div>
+        <div class="col">
+          <p class="text-uppercase"><strong>Crédito</strong></p>
+        </div>
+      </div>
+
+
+      <div class="row" *ngFor="let mvto of transaccion.movimientos; let i= index;">
+        <div class="col">
+           <p class="text-uppercase">{{[i+1]}}</p>
+        </div>
+        <div class="col">
+           <p class="text-uppercase">{{mvto.TerceroId}}</p>
+        </div>
+        <div class="col">
+           <p class="text-uppercase">{{mvto.CuentaId}}</p>
+        </div>
+        <div class="col">
+           <p class="text-uppercase">{{mvto.NombreCuenta}}</p>
+        </div>
+        <div class="col">
+           <p class="text-uppercase">{{mvto.Descripcion}}</p>
+        </div>
+        <div class="col" *ngIf="mvto.TipoMovimientoId==344">
+           <p class="text-uppercase">{{mvto.Valor}}</p>
+        </div>
+        <div class="col" *ngIf="mvto.TipoMovimientoId==345">
+           <p class="text-uppercase">0</p>
+        </div>
+        <div class="col" *ngIf="mvto.TipoMovimientoId==345">
+           <p class="text-uppercase">{{mvto.Valor}}</p>
+        </div>
+        <div class="col" *ngIf="mvto.TipoMovimientoId==344">
+           <p class="text-uppercase">0</p>
```

Comment:
Revisar primero los comentarios al .ts de este componente
```suggestion
        <div class="col" *ngIf="mvto.TipoMovimientoId == m_db">
           <p class="text-uppercase">{{mvto.Valor}}</p>
        </div>
        <div class="col" *ngIf="mvto.TipoMovimientoId == m_cr">
           <p class="text-uppercase">0</p>
        </div>
        <div class="col" *ngIf="mvto.TipoMovimientoId == m_cr">
           <p class="text-uppercase">{{mvto.Valor}}</p>
        </div>
        <div class="col" *ngIf="mvto.TipoMovimientoId == m_db">
           <p class="text-uppercase">0</p>
```
Una vez revisados, probar si se pueden usar directamente las constantes declaradas allá, para ver si se pueden ahorrar las variables `m_db` y `m_cr`